addons/block_code/code_generation/block_extension.gd

```diff
@@ -0,0 +1,10 @@
+@tool
+
+# FIXME: We should use some kind of prefix for these things. Is this okay?
+class_name EBC_BlockExtension
```

Comment:
Yes I love the idea of a EBC_ prefix for every class that needs a name. We started to follow this pattern in f3cbacb3 for internal scripts:

```
const BlockExtension = preload("res://addons/block_code/code_generation/block_extension.gd")
```

But this doesn't allow extending the class like in `extends EBC_BlockExtension`.

Reply:
Is that... a bike shed?! I feel like we either want underscores or camel case but not both. So, `EbcSomeClass` seems like the natural continuation of the camel case way.